tests/stratagus/test_depend.cpp

```diff
@@ -357,4 +360,5 @@ TEST_CASE("spell button depend")
 	CHECK(PrintDependencies(player, button) == requirements + "-upgradeMain\n");
 
 	Selected.clear();
+	CleanUnitTypes();
```

Comment:
`CleanDependencies();` seems missing, no?

Reply:
It did not change anything, but now it looks consistent.